test/performance_tests/Makefile

```diff
@@ -1,55 +0,0 @@
-
```

Comment:
Please return back each `makefile` for now. Not everybody has CMake. It would be no harm to leave this file as it is. Don't you think?

Reply:
You are completely right concerning the Makefiles